src/core/fixedsizealloc.c

```diff
@@ -294,7 +294,7 @@ void MVM_fixed_size_free(MVMThreadContext *tc, MVMFixedSizeAlloc *al, size_t byt
 #ifdef MVM_VALGRIND_SUPPORT
         if (RUNNING_ON_VALGRIND) {
             char command[128];
-            snprintf(&command, 128, "check_memory defined %p %d", dbg, bytes + 8);
+            snprintf(command, 128, "check_memory defined %p %lu", dbg, bytes + 8);
```

Comment:
Shouldn't that be %zu since bytes is a size_t?

Reply:
Guess so, `%ld` is what gcc suggested, but I made it `%lu` to be consistent with the format below. I can change both.

Reply:
Did that, but also changed `MVMFixedSizeAllocDebug->alloc_size` to be `size_t`.